### PR TITLE
Added code to handle projects.

### PR DESCRIPTION
**Pull request by christophluehr**
Hint: It seems as if there is a problem with the API on the Fastbill-side: searching for projectIds does not yield correct results (e.g. the wrong project is found).

### CODE REVIEW

**Comment by teiling88** [comment]
Thanks for your contribution! Looks good to me. 

I extended your PR for setProjectNumberFilter() because the guys from fastbill will extend it in this week :-)

If everything is ok for you I will merge your pull request.

File: src/Project/ProjectService.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    }
+
+    public function updateProject(ProjectEntity $entity): ProjectEntity
+    {
+        $this->checkErrors($this->validator->validateRequiredUpdateProperties($entity));
+
+        $this->xmlService->setService('project.update');
+        $this->xmlService->setData($entity->getXmlData());
+
+        echo $this->xmlService->getXml();
```
Comment (teiling88) [inline src/Project/ProjectService.php]: forgotten debug statement?

File: src/Project/ProjectService.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+     * @return ProjectEntity[]
+     */
+    public function getProject(ProjectSearchStruct $searchStruct): array
+    {
+        $this->xmlService->setService('project.get');
+        $this->xmlService->setFilters($searchStruct->getFilters());
+        $this->xmlService->setLimit($searchStruct->getLimit());
+        $this->xmlService->setOffset($searchStruct->getOffset());
+
+        echo $this->xmlService->getXml();
```
Comment (teiling88) [inline src/Project/ProjectService.php]: forgotten debug statement?

**Comment by teiling88** [comment]
Oh Mario from fastbill extended the search for ProjectNumber already. I removed your echo statements from your pr. 

If everything is ok - I will merge your pull request :-)

File: src/Project/ProjectService.php (hunk 1 above)
Comment (christophluehr) [reply src/Project/ProjectService.php]: True - bummer. Fixed.

File: src/Project/ProjectService.php (hunk 2 above)
Comment (christophluehr) [reply src/Project/ProjectService.php]: True - bummer. Fixed.

**Comment by teiling88** [comment]
Thanks for your contribution!

**Pull request merged**